lib/dialects/pgnative/index.js

```diff
@@ -0,0 +1,42 @@
+// PostgreSQL Native Driver (pg-native)
+// -------
+const Client_PG = require('../postgres');
+
+class Client_PgNative extends Client_PG {
+  acquireRawConnection() {
```

Comment:
How does this part differ from the PG version?

Reply:
This related to the comment here: https://github.com/knex/knex/pull/4327#issuecomment-788865473

The difference is limited to the call signature:

```javascript
      connection.connect(function (err, connection) {
```

vs.

```javascript
      connection.connect(function (err) {
```

Since the former signature looks to have been deprecated I'm happy to update the `postgres` driver to support the new syntax and remove this override if there are no objections (there seemed to be reservations in the original attempt).

Reply:
That would not require any changes on user end, right? If so, go for it :)

Reply:
Excellent. Swapped in most recent commit.